Basic/2DSpaceShooter/Assets/Scripts/RandomPositionPlayerSpawner.cs

```diff
@@ -43,9 +44,20 @@ private void Awake()
         networkManager.ConnectionApprovalCallback += ConnectionApprovalWithRandomSpawnPos;
     }
 
-    void ConnectionApprovalWithRandomSpawnPos(byte[] payload, ulong clientId, NetworkManager.ConnectionApprovedDelegate callback)
+    void ConnectionApprovalWithRandomSpawnPos(NetworkManager.ConnectionApprovalRequest request, NetworkManager.ConnectionApprovalResponse response)
```

Comment:
Do we need connection approval at all at this point? There's an option in the NetworkManager to disable it. Or do we have doc pointing to this? Why do we have a pending approval that doesn't do much? Could we point users to boss room for pending approval examples?

Reply:
The connection approval here is responsible for determining the player spawning positions. But the pending approval is just a test that I forgot to remove. I'll fix this

Reply:
player position can be set around player object spawn also, no need for connection approval for this.

Reply:
But I guess we already have an example of this in ClientDriven. Ok, if you could add some comments on this for position, that'd be great.